src/components/WfsLayer.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useState } from 'react'
+import L, { GeoJSONOptions } from 'leaflet'
+import { FeatureCollection } from 'geojson'
+
+import { useMapInstance, GeoJSON } from '@datapunt/react-maps'
+import fetchWithAbort from '../utils/fetchWithAbort'
+import { ZoomLevel } from '../types'
+import { MIN_ZOOM_LEVEL, MAX_ZOOM_LEVEL } from '../constants'
+
+const NO_DATA: FeatureCollection = {
+  type: 'FeatureCollection',
+  features: [],
+}
+
+interface Props {
+  url: string
+  options?: GeoJSONOptions
+  zoomLevel: ZoomLevel
+}
+
+const getBBox = (mapInstance: any) => {
```

Comment:
Nice! Didn't knew you could use `L.Util`. Perhaps we can move this to utils? And you can use the `Map` type from leaflet for the `mapInstance`

Reply:
done